encord/orm/workflow.py

```diff
@@ -0,0 +1,17 @@
+from collections import OrderedDict
+from enum import Enum
+
+from encord.orm import base_orm
+
+
+class WorkflowAction(Enum):
+    REOPEN = "reopen"
+    COMPLETE = "complete"
+
+
+class LabelWorkflowGraphNode:
+    pass
+
+
+class LabelWorkflowGraphNodePayload(base_orm.BaseORM):
```

Comment:
Not really, mostly for consistency and because it's serialisable to json by default. 
dataclasses.asdict(obj) should be equally fine, I suppose.
If we want to bring pydantic and match server-side approach I guess is a separate question. Totally happy to discuss, but would vote for not making anything like that in scope of this PR